src/main/java/software/amazon/awssdk/aws/greengrass/GeneratedAbstractCancelLocalDeploymentOperationHandler.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.awssdk.aws.greengrass;
+
+import software.amazon.awssdk.aws.greengrass.model.CancelLocalDeploymentRequest;
+import software.amazon.awssdk.aws.greengrass.model.CancelLocalDeploymentResponse;
+import software.amazon.awssdk.eventstreamrpc.OperationContinuationHandler;
+import software.amazon.awssdk.eventstreamrpc.OperationContinuationHandlerContext;
+import software.amazon.awssdk.eventstreamrpc.OperationModelContext;
+import software.amazon.awssdk.eventstreamrpc.model.EventStreamJsonMessage;
+
+public abstract class GeneratedAbstractCancelLocalDeploymentOperationHandler extends
```

Comment:
do we want/need this? Cloud deployments do not have an equivalent. Cloud deployments are canceled when a new deployment comes in.

Reply:
Discussed offline.

Today, when customer cancels a cloud deployment, they call`aws greengrassv2 cancel-deployment --deployment-id deploymentId` and we are trying to bring parity to that CLI experience, which will require a new CLI command.